src/surfaces/drawer/drawer.jsx

```diff
@@ -236,36 +236,36 @@ class Drawer extends React.Component {
 
     onClickOutside(event) {
         const {
-            onClickOutside,
+            shouldCloseOnClickOutside,
         } = this.props;
 
-        if (this.drawerContainerRef.contains(event.target) || !onClickOutside) {
+        if (this.drawerContainerRef.contains(event.target) || !shouldCloseOnClickOutside) {
             return;
         }
 
         this.onClose();
     }
 
     onClose() {
-        const { onClickOutside, onClose } = this.props;
+        const { shouldCloseOnClickOutside, onClose } = this.props;
 
-        if (onClickOutside) {
+        if (shouldCloseOnClickOutside) {
             document.removeEventListener('click', this.onClickOutside);
         }
 
         if (isFunction(onClose)) {
             onClose(...arguments); // eslint-disable-line prefer-rest-params
         } else {
-            console.warning('Drawer\'s onClose prop is required when using the prop onClickOutside'); // eslint-disable-line no-console
+            console.warning('Drawer\'s onClose prop is required when using the prop shouldCloseOnClickOutside'); // eslint-disable-line no-console
```

Comment:
NAB: We should adjust the ESLINT settings to allow `info`, `warning` and `error` maybe, like we did in `church-management` repo.

Reply:
copied from hc and pasted here.

Reply:
Nice, thanks!